client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -1,11 +1,18 @@
 @import "@wordpress/components/build-style/style";
 @import "@automattic/onboarding/styles/mixins";
-@import "@wordpress/base-styles/_breakpoints.scss";
-@import "@wordpress/base-styles/_mixins.scss";
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@wordpress/base-styles/mixins.scss";
 
 $gray-100: #101517;
 $gray-60: #50575e;
 $design-button-primary-color: rgb(17, 122, 201);
+$break-design-preview: 1024px;
```

Comment:
Time to update (or remove) this comment!

https://github.com/Automattic/wp-calypso/blob/28986240f916fc733830782a3589c62eae090deb/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx#L50-L51

Reply:
It's still being used [here](https://github.com/Automattic/wp-calypso/blob/28986240f916fc733830782a3589c62eae090deb/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx#L422) 🥲

Reply:
No I mean the comment, it still says `600px`